packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMedia.ts

```diff
@@ -271,4 +273,64 @@ describe('RemoteMedia', () => {
       }
     );
   });
+
+  describe('getEffectiveMaxFs()', () => {
+    it('returns maxFrameSize when it is greater than 0', () => {
+      remoteMedia.maxFrameSize = 1000;
+      
+      const result = remoteMedia.getEffectiveMaxFs();
+      
+      assert.strictEqual(result, 1000);
+    });
+
+    it('returns getMaxFs result when maxFrameSize is 0 and resolution is provided', () => {
+      remoteMedia.maxFrameSize = 0;
+      // remoteMedia was created with {resolution: 'medium'} in beforeEach
+      
+      const result = remoteMedia.getEffectiveMaxFs();
+      
+      // 'medium' resolution should map to 720p which is 3600
+      assert.strictEqual(result, 3600);
+    });
+
+    it('returns undefined when maxFrameSize is 0 and no resolution is provided', () => {
+      remoteMedia.maxFrameSize = 0;
+      // Create a new RemoteMedia without resolution option
+      const remoteMediaWithoutResolution = new RemoteMedia(fakeReceiveSlot, fakeMediaRequestManager);
+      
+      const result = remoteMediaWithoutResolution.getEffectiveMaxFs();
+      
+      assert.strictEqual(result, undefined);
+    });
+
+    it('prioritizes maxFrameSize over resolution option', () => {
+      remoteMedia.maxFrameSize = 500;
```

Comment:
nitpick: it might be nicer to call setSizeHint() from the test to set the `remoteMedia.maxFrameSize`, because that's what the app will actually do